app/controllers/api/v1/ecf_users_controller.rb

```diff
@@ -52,6 +52,9 @@ def email
 
       def users
         users = User.where(id: ParticipantProfile::ECF.joins(:teacher_profile).select("teacher_profiles.user_id"))
```

Comment:
@kicferk1 I'm a little concerned that there's not an `active` on the participant profiles or a `distinct` on the users - what's the intended behaviour here?

Reply:
Um, where would you like `distinct`? The whole thing is `User` query, I would hope that will give me distinct users by default. As in, `User.where(id: ["xxx", "xxx"]).count` gives me 1 when xxx is a valid user id.

As for `active` profiles, uhh. I think due to low impact the work on 'what to do with withdrawn participants on E&L' got de-prioritised. I think that ideally, we would add a field in here that says whether the user is active in the serialiser - so we can update our db - and return all users that have a profile, whether they are active or not.

Reply:
@xswoodsx might have an opinion on it too :)